### ISSUE TITLE
Fix height to parent div

### ISSUE THREAD

**Issue by Jhonb** [nicinabox/superslides#136]
Hi,

I would like to know how I can fix the height of the slider to the parent div.

Here's my html. http://pastebin.com/JbKgq1cD

My .slider div height = 450px:

**Comment by nicinabox** [comment]
`inherit_height_from` should be an element selector.

**Comment by Jhonb** [comment]
I tried too. Can you please give me an example?

Thanks!

---

Jhon Bauwens - +32 474 79 41 82 - jhon@faver.be - www.faver.be

**Comment by neridev** [comment]
Hello, this work for me:

examle JS code:
jQuery(document).ready(function($) {
        $('#slides').superslides({
            hashchange: false,
            pagination: true,
            play: 7000,
            animation: 'fade',
            animation_speed: 1000,
            inherit_height_from: slider,
        });
     });

example HTML code:
div id="slider"
       div id="slides"
            div class="slides-container"
            /div

example CSS code:
#slider {
     height: 450px;
}

**Comment by Jhonb** [comment]
Does not work for me… Images are not displayed. When I remove this line: inherit_height_from: slider,

It works again !

Le 16 août 2013 à 11:18, neri7dev notifications@github.com a écrit :

> Hello, this work for me:
> 
> examle JS code:
> jQuery(document).ready(function($) {
> $('#slides').superslides({
> hashchange: false,
> pagination: true,
> play: 7000,
> animation: 'fade',
> animation_speed: 1000,
> inherit_height_from: slider,
> });
> });
> 
> example HTML code:
> 
> example CSS code:
> #slider {
> height: 450px;
> }
> 
> —
> Reply to this email directly or view it on GitHub.

---

Jhon Bauwens - +32 474 79 41 82 - jhon@faver.be - www.faver.be

---

**Comment by nicinabox** [comment]
https://github.com/nicinabox/superslides/blob/0.6-stable/examples/constrained.html

**Comment by task1** [comment]
I was having the same issue and couldn't figure out what the deal was... I'm using Microsoft MVC 4 and Zurb Foundation. I seemed to have this issue in Chrome (not IE 10). I used a Nuget package to quickly add Foundation to my project using "responsive mvc" website as a quick reference for setup (http://www.responsivemvc.net/Foundation). The problem was this little piece of code that was included in the nuget package (the shared foundation layout)....

```
<script>
        document.write('<script src=' +
        ('__proto__' in {} ? '@Url.Content("~/scripts/zepto.js")' : '@Url.Content("~/scripts/jquery.js")') +
      '><\/script>')
    </script>
```

So Zepto was being spit out instead of a full fledged jQuery library. I got rid of the above code and simply added a reference to an actual jQuery library....fixed it. Maybe you're having a similar issue.
